src/sentry/grouptype/grouptype.py

```diff
@@ -0,0 +1,109 @@
+from dataclasses import dataclass
+
+from sentry.event_manager import DEFAULT_GROUPHASH_IGNORE_LIMIT
+from sentry.types.issues import GroupCategory
+
+_group_type_registry = {}
+
+
+@dataclass(frozen=True)
+class GroupType:
+    type_id: int
+    slug: str
+    description: str
+    category: int
+    ignore_limit: int = DEFAULT_GROUPHASH_IGNORE_LIMIT
```

Comment:
I think that the ignore limit probably belongs as part of the policy layer so that it can be dynamically changed

Reply:
we can remove it later though? might take some time to spin up

Reply:
That's reasonable, especially since it's currently in code at the moment.